sms_statistics_service/src/firestore_wrapper.py

```diff
@@ -47,14 +62,23 @@ def update_sms_stats(self, project_name, sms_stats_batch):
         :type sms_stats_batch: dict of str -> src.data_models.SMSStats
         """
         log.info(f"Batch updating {len(sms_stats_batch)} SMS stats for project {project_name}...")
+        batch = self.client.batch()
+        batch_counter = 0
+        total_progress = 0
```

Comment:
I think total_counter is more consistent?

Reply:
changed